.github/workflows/actions.yml

```diff
@@ -83,19 +83,21 @@ jobs:
         uses: actions/checkout@v3
       - name: run-app-bg
         run: make run-app-bg
-      - name: populate DB
+      - name: populate DB & take screenshots
```

Comment:
Thanks for this change! To me, this suggests we should clean and merge https://github.com/conbench/conbench/pull/592 instead of letting it go. I like the fake data in that PR better than the existing fake data, especially for these history plots. :)

Reply:
Yeah absolutely. Had the very same thought, but want do this this in separate PRs, step by step. I look at the current populate_local_conbench.py as very temporary.

My plan is certainly to land https://github.com/conbench/conbench/pull/592, or a derivative thereof, asap.